app/features/config/SelectWowDir.tsx

```diff
@@ -20,6 +20,21 @@ function getDefaultPath() {
   return '';
 }
 
+function validateSelectedPath(path: String) {
```

Comment:
String should be lower case. Primitives are lower case - uppercase String means that its wrapped in a wrapper object